### PR TITLE
Form for creating and naming a new project

### PR DESCRIPTION
**Pull request by AtbTheGreat**

### CODE REVIEW

File: app/Routes.js [hunk 1]
```diff
@@ -3,12 +3,14 @@ import React from 'react';
...
-import HomePage from './containers/HomePage';
+//import HomePage from './containers/HomePage';
+import ProjectSetupPage from './containers/ProjectSetupPage';
 
 export default () => (
   <App>
     <Switch>
-      <Route path={routes.HOME} component={HomePage} />
+      <Route path={routes.PROJECT_SETUP} component={ProjectSetupPage} />
+      <Route path={routes.HOME} component={ProjectSetupPage} />
```
Comment (drewatk) [inline app/Routes.js]: Change this back to HomePage

File: app/components/Home.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 // @flow
 import React from 'react';
-import ProseEditor from './ProseEditor';
+//import ProseEditor from './ProseEditor';
+import ProjectSetup from './ProjectSetup';
 
 const Home = () => {
-  return <ProseEditor />;
+  return <ProjectSetup />;
```
Comment (drewatk) [inline app/components/Home.js]: And change this back to ProseEditor

File: app/components/ProjectSetup/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ProjectList from './ProjectList';
+import NewProjectForm from './NewProjectForm';
+
+const ProjectSetup = () => {
+  const projects = [
+    { name: 'project 1' },
```
Comment (drewatk) [inline app/components/ProjectSetup/index.js]: @marsha88 @vritant24 Any other properties that Anna should write in to display these projects? Will these be sorted?

Last recently used possibly?

**Comment by drewatk** [comment]
Yep. Haven't made that part yet =D

File: app/components/ProjectSetup/index.js (hunk 3 above)
Comment (vritant24) [reply app/components/ProjectSetup/index.js]: For now we can keep it alphabetical. Recently used can be a later improvement.
I don't think anything else needs to be shown for now. Project name is good enough

**Comment by marsha88** [comment]
Does this PR completely overwrite #14?

**Comment by drewatk** [comment]
I think we can close #14, this is branched off of it

File: app/components/Home.js [hunk 4]
```diff
@@ -1,6 +1,7 @@
 // @flow
 import React from 'react';
 import ProseEditor from './ProseEditor';
+//import ProjectSetup from './ProjectSetup';
```
Comment (drewatk) [inline app/components/Home.js]: Unnecessary comment

**Pull request merged**